frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -357,17 +358,60 @@ private void downloadLicenseInfo(ResourceRequest request, ResourceResponse respo
             final LicenseInfoService.Iface licenseInfoClient = thriftClients.makeLicenseInfoClient();
             final User user = UserCacheHolder.getUserFromRequest(request);
             Project project = thriftClients.makeProjectClient().getProjectById(projectId, user);
+            includeReleasesByChosenRelationship(request, releaseIdsToSelectedAttachmentIds);
             LicenseInfoFile licenseInfoFile = licenseInfoClient.getLicenseInfoFile(project, user, outputGenerator,
                     releaseIdsToSelectedAttachmentIds, excludedLicensesPerAttachmentId, externalIds);
             saveLicenseInfoAttachmentUsages(project, user, selectedAttachmentIdsWithPath,
-                    excludedLicensesPerAttachmentIdWithPath);
+                    excludedLicensesPerAttachmentIdWithPath, getSelectedReleaseRationships(request));
```

Comment:
`getSelectedReleaseRationships(request)` is also called in `includeReleasesByChosenRelationship(request, releaseIdsToSelectedAttachmentIds);`  to get same value. Maybe it can be called once and its value can be reused.

Reply:
Done